src/SampleWebApp/Views/BooksMvc/Edit.cshtml

```diff
@@ -0,0 +1,65 @@
+@model SampleWebApp.Models.Book
+
+@{
+    ViewData["Title"] = "Edit";
+}
+
+<h2>Edit</h2>
+
+<form asp-action="Edit">
+    <div class="form-horizontal">
+        <h4>Book</h4>
+        <hr />
+        <div asp-validation-summary="ValidationSummary.ModelOnly" class="text-danger"></div>
+        <div class="form-group">
+            <label asp-for="AuthorID" class="control-label col-md-2">AuthorID</label>
```

Comment:
This was a bug in the scaffolding template's tag helper. Its addressed as part of aspnet/Scaffolding/issues/189